effectiveOfficeBackend/src/main/kotlin/office/effective/Application.kt

```diff
@@ -0,0 +1,17 @@
+package office.effective
+
+import io.ktor.server.application.*
+import io.ktor.server.engine.*
+import io.ktor.server.netty.*
+import office.effective.plugins.*
+
+fun main() {
+    embeddedServer(Netty, port = 8080, host = "0.0.0.0", module = Application::module)
```

Comment:
Also if you have a constant predefined port, it will be necessary to document it for ops, i. e. in Dockerfile it might look like

```dockerfile
EXPOSE 8080
```